src/IRMatch.h

```diff
@@ -239,7 +239,7 @@ struct WildConstInt {
             halide_scalar_value_t val;
             halide_type_t type;
             state.get_bound_const(i, val, type);
-            return op->type == type && value == val.u.i64;
+            return e.type == type && value == val.u.i64;
```

Comment:
eep. this looks like a scary pre-existing bug?

Reply:
Yes. Will pull into separate PR.